src/main/scala/is/hail/expr/HailRep.scala

```diff
@@ -85,6 +86,10 @@ trait HailRepFunctions {
     def typ = TArray(hrt.typ)
   }
 
+  implicit def wrappedArrayHr[T](implicit hrt: HailRep[T]) = new HailRep[mutable.WrappedArray[T]] {
```

Comment:
Good point, I can make `HailRep` contra variant in its argument which allows me to use `HailRep[IndexedSeq[T]]` anywhere I need a subtype of `IndexedSeq[T]`.

I have a tendency avoid co/contra-variance because it always seems to bite me down the road.